### PR TITLE
Fix $group operator unicode issue

### PR DESCRIPTION
**Pull request by polosatyi**

### CODE REVIEW

File: mongomock/collection.py [hunk 1]
```diff
@@ -1235,7 +1235,7 @@ def _get_group_id_fields(self, keys, parent=''):
 
         if isinstance(keys, dict):
             for k, v in keys.items():
-                if isinstance(v, str):
+                if isinstance(v, str) or isinstance(v, unicode):
```
Comment (srinivasreddy) [inline mongomock/collection.py]: rather `isinstance(v, six.text_type)`

Comment (mdomke) [inline mongomock/collection.py]: What @srinivasreddy says!

**Comment by mdomke** [comment]
Ping @polosatyi! Do you want to include the suggestions of @srinivasreddy so that I can merge your pull-request?

**Comment by mdomke** [comment]
Hey @polosatyi! In #236 @srinivasreddy already implemented the suggestions to your pull-request so I'm closing this one. Thank you anyhow for the support.

**Pull request closed without merging**